### PR TITLE
Validate SARJson

### PR DESCRIPTION
**Pull request by anu-prithviraj**
This feature branch is used to implement the validation of SARJson against the schema written according to draft 6 specifications and include:

1. json_schema_validate.py (main script)
2. test_json_schema_validate.py (pytest script)
3. InputData Folder 
     1. valid_json.json
     2. invalid_json.json
4. Schema Folder
    1. SARJSchema.json

### CODE REVIEW

File: test_json_schema_validate.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+the given schema.
+"""
+
+from json_schema_validate import ValidateJson
+import pytest
+import pathlib
+import os
+
+get_abs_path = pathlib.Path().absolute()
+valid_json_path = os.path.join(get_abs_path,'InputData','valid_json.json')
```
Comment (VayneSilvermyst) [inline test_json_schema_validate.py]: So, here, you dont really have to mix pathlib and os.join. you can join paths just by 
valid_json_path = get_abs_path/"InputData/valid_json.json"

So you are reducing one extraimport

File: json_schema_validate.py [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+            jsonschema.validate(instance= data, schema=schema_to_match)
+            flag = True
+            print("\n Given JSON is valid")
+        except jsonschema.exceptions.ValidationError as Verr:
+            print(Verr)
+            flag = False
+            print("\n Given JSON is not valid")
+        return flag
+        
+def cmd_line_args():
```
Comment (VayneSilvermyst) [inline json_schema_validate.py]: This function returns the parser right? so, may be name it something like, make_parser or create_parser or something as that will be more appropriate

File: Schema/SARJSchema.json [hunk 3]
```diff
@@ -0,0 +1,302 @@
+{
```
Comment (VayneSilvermyst) [inline Schema/SARJSchema.json]: Please delete all of Illumina jsons from your repo

File: InputData/sample-sarj.json [hunk 4]
```diff
@@ -0,0 +1,425 @@
+{
```
Comment (VayneSilvermyst) [inline InputData/sample-sarj.json]: for test use test data with no information, Please delete all illumina files from this repo

File: json_schema_validate.py [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+    def __init__(self, jdata, schema):
+
+        """
+        class constructor
+        """
+
+        self.jdata = jdata
+        self.schema = schema
+       
+    def load_json(self, file_name):
```
Comment (VayneSilvermyst) [inline json_schema_validate.py]: This looks nice

File: test_json_schema_validate.py [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+    Input:
+        valid json, schema to match
+    Output:
+        Return a True   
+    """
+
+    valid_json_object = ValidateJson(valid_json_path, schema_path)
+    output = valid_json_object.validate_json()
+    assert output == True 
+
```
Comment (VayneSilvermyst) [inline test_json_schema_validate.py]: you hve to test the load json functionality as well

**Review by VayneSilvermyst — CHANGES_REQUESTED** [review]
You have to remove all illumina files from the repo.And some minor comments as well

File: json_schema_validate.py (hunk 2 above)
Comment (anu-prithviraj) [reply json_schema_validate.py]: Function name updated.

File: test_json_schema_validate.py (hunk 6 above)
Comment (anu-prithviraj) [reply test_json_schema_validate.py]: Done.

**Pull request merged**